app/assets/javascripts/contributors_readmore.js

```diff
@@ -0,0 +1,38 @@
+var readMoreShow = (function(){
+    $('.read-more-show').on('click', function(e){
+        $(this).fadeOut(400, function(){
+            $(this).next('.read-more-content').removeClass('hide').slideDown(1000);
+            $(this).addClass('hide');
+        });
+        e.preventDefault();
```

Comment:
Hmm I thought it was needed for all click events to prevent the page from automatically causing the event to occur whilst loading. It doesn't seem to be a problem here while I checked in my local server, so I'll remove it.